common/src/main/java/com/wynntils/wc/custom/item/properties/HorseProperty.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.wc.custom.item.properties;
+
+import com.wynntils.mc.utils.ItemUtils;
+import com.wynntils.wc.custom.item.WynnItemStack;
+import java.util.List;
+
+public class HorseProperty extends ItemProperty {
+
+    private int xp;
+    private int level;
+    private int maxLevel;
+    private int tier;
+
+    public HorseProperty(WynnItemStack item) {
+        super(item);
+
+        // Parse xp, level, max level, and tier
+        List<String> lore = ItemUtils.getLore(item);
+
+        tier = Integer.parseInt(lore.get(0).substring(7));
+        level = Integer.parseInt(lore.get(1).substring(9, lore.get(1).indexOf('/')));
+        maxLevel = Integer.parseInt(lore.get(1).substring(lore.get(1).indexOf('/') + 1));
+        xp = Integer.parseInt(lore.get(4).substring(6, lore.get(4).indexOf('/')));
```

Comment:
Overall I prefer regexp for this kind of matching. It is clearer what is going on, and if the format changes it will fail correctly, instead of extracting random subparts of the strings.

But if you don't feel like redoing it as a regex I'm (reluctantly) okay with it.